spec/std/int_spec.cr

```diff
@@ -297,7 +297,7 @@ describe "Int" do
   describe "step" do
     it "steps through limit" do
       passed = false
-      1.step(1) { |x| passed = true }
+      1.step(by: 1) { |x| passed = true }
```

Comment:
This is the test that hangs travis. This should be `to: 1`, otherwise it iterates infinitely